terraform/environments/digital-prison-reporting/temporary_domain_refresh_jobs.tf

```diff
@@ -0,0 +1,282 @@
+  execution_class             = "FLEX"
+  worker_type                 = local.refresh_job_worker_type
+  number_of_workers           = local.refresh_job_num_workers
+  max_concurrent              = 64
+  region                      = local.account_region
+  account                     = local.account_id
+  log_group_retention_in_days = 1
+
+  tags = merge(
+    local.all_tags,
+    {
+      Name          = "${local.domain_refresh_prisoner_prisoner}"
+      Resource_Type = "Glue Job"
+      Jira          = "DPR2-193"
+    }
+  )
+
+  arguments = {
+    "--extra-jars"                   = local.glue_jobs_latest_jar_location
+    "--class"                        = "uk.gov.justice.digital.job.DomainRefreshJob"
+    "--datalake-formats"             = "delta"
+    "--dpr.aws.dynamodb.endpointUrl" = "https://dynamodb.${local.account_region}.amazonaws.com"
+    "--dpr.aws.kinesis.endpointUrl"  = "https://kinesis.${local.account_region}.amazonaws.com"
+    "--dpr.aws.region"               = local.account_region
+    "--dpr.curated.s3.path"          = "s3://${module.s3_curated_bucket.bucket_id}"
+    "--dpr.domain.registry"          = "${local.project}-domain-registry-${local.environment}"
+    "--dpr.domain.target.path"       = "s3://${module.s3_domain_bucket.bucket_id}"
+    "--dpr.domain.catalog.db"        = module.glue_data_domain_database.db_name
+    "--dpr.log.level"                = local.refresh_job_log_level
+    "--dpr.domain.name"              = "prisoner"
+    "--dpr.domain.table.name"        = "prisoner"
+    "--dpr.domain.operation"         = "sync"
+  }
+}
+
+# =======================================
+# SCHEDULES
+# =======================================
+
+# Establishment Establishment Schedule
+resource "aws_glue_trigger" "temp_domain_refresh_establishment_establishment" {
+  name     = "${local.domain_refresh_establishment_establishment}-trigger"
+  schedule = "cron(*/15 06-20 * * MON-FRI *)"
+  type     = "SCHEDULED"
+
+  actions {
+    job_name = module.glue_temp_refresh_job_establishment_establishment.name
+  }
+}
+
+
+# Establishment Living Unit Schedule
+resource "aws_glue_trigger" "temp_domain_refresh_establishment_living_unit" {
+  name     = "${local.domain_refresh_establishment_living_unit}-trigger"
+  schedule = "cron(*/15 06-20 * * MON-FRI *)"
+  type     = "SCHEDULED"
+
+  actions {
+    job_name = module.glue_temp_refresh_job_establishment_living_unit.name
+  }
+}
+
+
+# Movements Movements Schedule
+resource "aws_glue_trigger" "temp_domain_refresh_movements_movements" {
+  name     = "${local.domain_refresh_movements_movements}-trigger"
+  schedule = "cron(*/15 06-20 * * MON-FRI *)"
+  type     = "SCHEDULED"
+
+  actions {
+    job_name = module.glue_temp_refresh_job_movements_movements.name
+  }
+}
+
+
+
+# Prisoner Prisoner Schedule
+resource "aws_glue_trigger" "temp_domain_refresh_prisoner_prisoner" {
+  name     = "${local.domain_refresh_prisoner_prisoner}-trigger"
+  schedule = "cron(*/15 06-20 * * MON-FRI *)"
```

Comment:
In the Jira ticket it mentions the jobs will run between 06:00-18:00. 
Is it intentional here to have it running until 20:00?

Reply:
I've made the change
0/15 until 19 (so it runs until 19:45)
Changed the ticket to reflect the change from 6pm to 8pm